DEVELOP.md

```diff
@@ -30,6 +27,22 @@ python3 -m pip install --user --upgrade twine
 python3 -m twine upload dist/*
 ```
 
+### R
+
+From the command prompt, navigate to `src/r`. Open R and run:
```

Comment:
I am still confused by your use case -- are you describing Windows? If so this is incomplete as there is no t copying in the licence file

Reply:
This is taken from your original commit. I also see the same description in the python section.
You are right, as I did not copy the LICENSE file.

As for the LICENSE file for the **R package**, I think a new license is needed here to denote the role of me and  @banfelder. What do you think?

Also, I get a little bit confused by what the **Deployment** means here. Does it solely aim to provide instructions on how to install the R/python library?

Reply:
> As for the LICENSE file for the R package, I think a new license is needed here to denote the role of me and @banfelder. What do you think?

I would prefer to have a single licence and modify that as this refers to Psychrolib and not just the port. @didierthevenard what do you think? This means that we would add the name of the other contributors to the license at https://github.com/psychrometrics/psychrolib/blob/master/LICENSE.txt, otherwise we can simply change to something like `<YEAR> The PsychroLib Contributors`

Also, I get a little bit confused by what the Deployment means here. Does it solely aim to provide instructions on how to install the R/python library?

Deployment is meant to be something that you as a developer needs to do if you want to further develop or manually install the library -- that is to also generate the artefacts like `man`.  For Fortran etc, there is no need for this as you can simply copy and paste the source file in and include it in your code -- this is why there is no mention.